.github/workflows/python-app.yml

```
@@ -32,3 +32,8 @@ jobs:
 
       - name: Run linting and tests
         run: make ci
+      
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)